2023-04-02 14:58 错误信息 HTTPSConnectionPool(host='bluearchive-cn.com', port=443): Max retries exceeded with url: /api/pre-reg/stats (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025CDB9D49A0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))
2023-04-02 14:59 错误信息 HTTPSConnectionPool(host='bluearchive-cn.com', port=443): Max retries exceeded with url: /api/pre-reg/stats (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001D78D7349A0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))
2023-04-02 15:00 错误信息 HTTPSConnectionPool(host='bluearchive-cn.com', port=443): Max retries exceeded with url: /api/pre-reg/stats (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001D78D7348B0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))
2023-04-02 15:01 错误信息 'gbk' codec can't decode byte 0xae in position 52: illegal multibyte sequence
